### ISSUE TITLE
update BLS entries on RH-based distros

### ISSUE THREAD

**Issue by alexsander-souza** [canonical/packer-maas#191]
as discussed in https://discourse.maas.io/t/rocky-9-3-deployment/7744

we should stop disabling the BLS entries in grub (`GRUB_ENABLE_BLSCFG=false`) and include proper support in Curtin.

**Comment by alexsander-souza** [comment]
Curtin bug: https://bugs.launchpad.net/curtin/+bug/2047863

**Comment by easpeagle** [comment]
Yeah... piling on here... I've deployed some RL 9.4 systems that end up having this issue. Grub is reading the grub.cfg from down in /boot/efi/EFI/rocky/grub.cfg ... but the system is building the grub menu in /boot/grub2/grub.cfg when new kernels are installed.

/etc/default/grub definitely shows "GRUB_ENABLE_BLSCFG=false" ... what's the right way to retrofit my system so that it works properly for BLS?

**Comment by alexsander-souza** [comment]
> /etc/default/grub definitely shows "GRUB_ENABLE_BLSCFG=false" ... what's the right way to retrofit my system so that it works properly for BLS?

try running `grub2-mkconfig --update-bls-cmdline -o /boot/grub2/grub.cfg`

**Comment by easpeagle** [comment]
> > /etc/default/grub definitely shows "GRUB_ENABLE_BLSCFG=false" ... what's the right way to retrofit my system so that it works properly for BLS?
> 
> try running `grub2-mkconfig --update-bls-cmdline -o /boot/grub2/grub.cfg`

Okay... what exactly is the `--update-bls-cmdline` option supposed to do? I don't see any change to `/etc/default/grub` ... and I suspect my system is still going to look for the grub.cfg in `/boot/efi/EFI/rocky/grub.cfg` ... I feel like perhaps grub needs to be reinstalled to look for its `grub.cfg` in `/boot/grub2/grub.cfg` ...
